Entities/DiagonalWingedStrawberry.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Celeste.Mod.Entities;
+using Microsoft.Xna.Framework;
+using Monocle;
+using MonoMod.Cil;
+using MonoMod.RuntimeDetour;
+using System;
+using System.Reflection;
+
+namespace Celeste.Mod.SpringCollab2020.Entities {
+    [CustomEntity("SpringCollab2020/diagonalWingedStrawberry")]
+    [RegisterStrawberry(true, false)]
+    class DiagonalWingedStrawberry : Strawberry {
+        public DiagonalWingedStrawberry(EntityData data, Vector2 offset, EntityID gid) : base(FixData(data), offset, gid) {
+            OriginalOnDash = typeof(Strawberry).GetMethod("OnDash", BindingFlags.Instance | BindingFlags.NonPublic | BindingFlags.InvokeMethod);
+
+            // Components.ToArray() is used because the map fails to load when using merely "Components" for some reason.
```

Comment:
That's because you're trying to remove items from the collection you're currently modifying. In contrast, `ToArray` creates a copy that you're iterating through, which will be untouched while you're removing things from `Components`.

Reply:
Ah, alright 👍